src/routes/(app)/news/SmallArticleCard.svelte

```diff
@@ -1,91 +1,62 @@
 <script lang="ts">
-  import MarkdownBody from "$lib/components/MarkdownBody.svelte";
-  import TagChip from "$lib/components/TagChip.svelte";
-  import type { Article as ArticleType, Member, Tag } from "@prisma/client";
-  import type { SuperValidated } from "sveltekit-superforms";
-  import AuthorSignature from "$lib/components/AuthorSignature.svelte";
-  import LikeButton from "./LikeButton.svelte";
-  import LikersList from "./LikersList.svelte";
-  import type { AuthorOption } from "./articles";
-  import type { LikeSchema } from "./likes";
+  import DOMPurify from "isomorphic-dompurify";
+  import { marked } from "marked";
+  import { getFullName } from "$lib/utils/client/member";
+  import type { Article } from "./articles";
 
-  export let hideBody = false;
-  export let onTagClick: ((tag: Tag) => void) | null = null;
-  export let article: ArticleType;
-  export let author: AuthorOption;
-  export let tags: Tag[];
-  export let likers: Member[];
-  export let commentCount: number;
-  export let likeForm: SuperValidated<LikeSchema>;
+  export let article: Article;
 </script>
 
-<article
-  class="ease my-4 rounded-lg p-6 shadow-2xl ring-neutral-700 transition md:ring-1 md:hover:scale-[1.01]"
->
-  <a href="/news/{article.slug}">
-    <h2 class="my-3 break-words text-2xl font-bold">{article.header}</h2>
-  </a>
-
-  <div class="flex flex-row justify-between">
-    <p class="text-right text-xs text-gray-500">
-      {article.publishedAt?.toLocaleDateString(["sv"])}
-      ,
-      {article.publishedAt?.toLocaleTimeString(["sv"], {
-        hour: "2-digit",
-        minute: "2-digit",
-      })}
-    </p>
-  </div>
-
-  {#if !hideBody}
-    <div class="my-3 flex flex-row items-start gap-2">
-      <MarkdownBody
-        body={article.body.slice(0, 400)}
-        class="line-clamp-4 text-ellipsis"
-      />
-      {#if article.imageUrl}
-        <figure class="flex-1">
-          <img
-            class="rounded-2xl"
-            src={article.imageUrl}
-            alt={article.header}
-          />
-        </figure>
-      {/if}
+<div class="flex flex-col overflow-hidden rounded-xl bg-base-200">
+  <div class="flex justify-between gap-4 bg-base-300 p-4">
+    <div class="flex flex-wrap gap-2">
+      {#each article.tags as tag}
+        <span class="badge text-xs text-neutral-400">{tag.name}</span>
+      {/each}
```

Comment:
An article without tags has the height of this bar lower. Would be nice if they were the same height

Reply:
Properly solving that seems tedious 😑. An magic number fix is easy enough though and results in this:

![image](https://github.com/Dsek-LTH/web/assets/26229521/836ab30b-8240-47e0-9bda-79cb58043ccc)

Reply:
Fixed now